config/containers/resource_constraints.md

```diff
@@ -77,7 +77,7 @@ Most of these options take a positive integer, followed by a suffix of `b`, `k`,
 
 | Option                 | Description                                                                                                                                                                                                                                                                                                                                                                                      |
 |:-----------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `-m` or `--memory=`    | The maximum amount of memory the container can use. If you set this option, the minimum allowed value is `6m` (6 megabyte).                                                                                                                                                                                                                                                                      |
+| `-m` or `--memory=`    | The maximum amount of memory the container can use. If you set this option, the maximum allowed value is `6m` (6 megabyte).                                                                                                                                                                                                                                                                      |
```

Comment:
The intent was to describe that values lower than `6m` are not allowed (e.g., setting `--memory=1m` will produce an error).

```bash
docker run --rm --memory=1m alpine

docker: Error response from daemon: Minimum memory limit allowed is 6MB.
See 'docker run --help'.
```


@usha-mandya perhaps you have a suggestion to better phrase that?

Reply:
@thaJeztah  I think the text "minimum allowed value" is pretty self-explanatory. Perhaps we could clarify by adding  "If you use this option, set the value to at least 6m (6 megabytes)."